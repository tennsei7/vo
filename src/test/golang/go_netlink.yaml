general:
  stop_time: 10s

network:
  graph:
    type: 1_gbit_switch

hosts:
  host:
    network_node_id: 0
    processes:
    - path: ./test_netlink
      start_time: 1s
