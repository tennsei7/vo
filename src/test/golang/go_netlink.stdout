ip+net 127.0.0.1/8
ip+net 11.0.0.1/24
